osf/models/mixins.py

```diff
@@ -758,13 +1030,17 @@ def check_subject_perms(self, auth):
         AbstractNode = apps.get_model('osf.AbstractNode')
         Preprint = apps.get_model('osf.Preprint')
         CollectionSubmission = apps.get_model('osf.CollectionSubmission')
+        DraftRegistration = apps.get_model('osf.DraftRegistration')
 
         if isinstance(self, AbstractNode):
             if not self.has_permission(auth.user, ADMIN):
                 raise PermissionsError('Only admins can change subjects.')
         elif isinstance(self, Preprint):
             if not self.has_permission(auth.user, WRITE):
                 raise PermissionsError('Must have admin or write permissions to change a preprint\'s subjects.')
+        elif isinstance(self, DraftRegistration):
+            if not self.has_permission(auth.user, WRITE):
```

Comment:
Should this be admin or write? I don't think we've decided.

Reply:
Changed to admin for consistency with current workflow, but feel free to modify.